### PR TITLE
[skip ci] rename Source to Sources

### PR DESCRIPTION
**Pull request by repjarms**
files with only comment changes

AssistantV1 changes

AssistantV2 changes

CompareComplyV1 changes

DiscoveryV1 changes

DiscoveryV2 changes

LT V3, NLC V1, NLU V1, PI V3 changes

STT V1, TTS V1, Vis Rec V3/V4 changes

changelog and xcode project file changes

all current hand edits

discovery v2 remove unused models

rename migration guide and fix changelog formatting

chore: add vis rec v4 hand edit and pi deprecation warning

update .gitignore and xcode project file

chore: update tests and regenerate

docs(readme): document setting global transaction id

Update README.md

Update README.md

ci: temporarily remove integration testing

docs: Add announcements section and pi deprecation

docs: add a known issues section describing Carthage and XCode 12 issues

test of skipping ci [skip ci]

[docs] Revise PI deprecation, add VisRec deprecation

chore: update gateway urls

chore: update podspecs

fix(rebase): move sources back

### Summary

Provide a general description of the code changes in your pull
request... were there any bugs you had fixed? If so, mention them. If
these bugs have open GitHub issues, be sure to tag them here as well,
to keep the conversation linked together.

### Other Information

If there's anything else that's important and relevant to your pull
request, mention that information here.

Thanks for contributing to the Watson Developer Cloud!

### CODE REVIEW

File: Source/VisualRecognitionV4/VisualRecognition.swift [hunk 1]
```diff
@@ -161,17 +181,16 @@ public class VisualRecognition {
...
-        if let csvCollectionIDsData = collectionIDs.joined(separator: ",").data(using: .utf8) {
-            multipartFormData.append(csvCollectionIDsData, withName: "collection_ids")
+        for item in collectionIDs {
+            if let itemData = item.data(using: .utf8) {
+                multipartFormData.append(itemData, withName: "collection_ids")
+            }
         }
-
-        // HAND EDIT: join features into CSV string
-        if let csvFeaturesData = features.joined(separator: ",").data(using: .utf8) {
```
Comment (repjarms) [inline Source/VisualRecognitionV4/VisualRecognition.swift:171]: this hand edit needs to be maintained

**Review by repjarms — COMMENTED** [review]
It looks like there are a number of hand edits that need to be retained across this release @apaparazzi0329

File: Source/ToneAnalyzerV3/ToneAnalyzer.swift [hunk 2]
```diff
@@ -167,22 +187,19 @@ public class ToneAnalyzer {
     public func tone(
         toneContent: ToneContent,
         sentences: Bool? = nil,
+        tones: [String]? = nil,
```
[the remark this answers is not in the document]
Comment (apaparazzi0329) [reply Source/ToneAnalyzerV3/ToneAnalyzer.swift:190]: This is a deprecated parameter that has no effect on the response but is still accepted for backward-compatibility purposes

[the remark this answers is not in the document]
Comment (apaparazzi0329) [reply Source/ToneAnalyzerV3/ToneAnalyzer.swift:190]: https://cloud.ibm.com/apidocs/tone-analyzer?code=node#gettone

[the remark this answers is not in the document]
Comment (apaparazzi0329) [reply Source/ToneAnalyzerV3/ToneAnalyzer.swift:190]: https://cloud.ibm.com/apidocs/tone-analyzer?code=node#gettone

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,276 @@
...
+### VisualRecognitionV3 Instance
+* No changes
+
+### VisualRecognitionV3 Models
+* No changes
+
+### VisualRecognitionV4 Instance
+* `createCollection()`: `trainingStatus` parameter added
+
+* `updateCollection()`: `trainingStatus` parameter added
```
Comment (mediumTaj) [inline CHANGELOG.md:254]: Training status should be removed from createCollection and updateCollection - these should be readonly. need to address this in the generator but please remove for now

File: Source/VisualRecognitionV3/VisualRecognition.swift [hunk 4]
```diff
@@ -13,22 +13,41 @@
...
     /// The base URL to use when contacting the service.
     public var serviceURL: String? = "https://api.us-south.visual-recognition.watson.cloud.ibm.com"
 
+    /// Release date of the API version you want to use. Specify dates in YYYY-MM-DD format. The current version is
+    /// `2018-03-19`.
+    public var version: String
+
     /// Service identifiers
-    internal let serviceName = "WatsonVisionCombined"
+    public static let defaultServiceName = "watson_vision_combined"
```
Comment (mediumTaj) [inline Source/VisualRecognitionV3/VisualRecognition.swift:48]: Is this the correct default service name?

File: Source/VisualRecognitionV4/VisualRecognition.swift [hunk 5]
```diff
@@ -13,22 +13,41 @@
...
     /// The base URL to use when contacting the service.
     public var serviceURL: String? = "https://api.us-south.visual-recognition.watson.cloud.ibm.com"
 
+    /// Release date of the API version you want to use. Specify dates in YYYY-MM-DD format. The current version is
+    /// `2019-02-11`.
+    public var version: String
+
     /// Service identifiers
-    internal let serviceName = "WatsonVisionCombined"
+    public static let defaultServiceName = "watson_vision_combined"
```
Comment (mediumTaj) [inline Source/VisualRecognitionV4/VisualRecognition.swift:48]: is this the correct default service name? also check assistant, nlu, compare comply. not sure what this should be for swift

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,4 +1,276 @@
...
+`AnalysisResults`
+* `metadata` type changed from `AnalysisResultsMetadata?` to `FeaturesResultsMetadata?`
+
+`Features`
+* `metadata` type changed from `MetadataOptions?` to `[String: JSON]?`
+
+### PersonalityInsights Instance
+* `profile()`: csvHeaders parameter added
+
+* `profileAsCSV()`: return type changed from `WatsonResponse<String>?` to `WatsonResponse<Data>?`
```
Comment (mediumTaj) [inline CHANGELOG.md:200]: added deprecation notice

File: Source/AssistantV1/Assistant.swift [hunk 7]
```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2018, 2020.
```
[the remark this answers is not in the document]
Comment (apaparazzi0329) [reply Source/AssistantV1/Assistant.swift:2]: Resolved

File: Source/TextToSpeechV1/TextToSpeech.swift [hunk 8]
```diff
@@ -357,47 +370,22 @@ public class TextToSpeech {
...
-        request.response { (response: WatsonResponse<Data>?, error: WatsonError?) in
-            var response = response
-            guard let data = response?.result else {
-                completionHandler(response, error)
-                return
-            }
-            if accept?.lowercased().contains("audio/wav") == true {
-                // repair the WAV header
-                var wav = data
-                guard WAVRepair.isWAVFile(data: wav) else {
```
[the remark this answers is not in the document]
Comment (apaparazzi0329) [reply Source/TextToSpeechV1/TextToSpeech.swift:369]: My testing has shown that we no longer need this

**Review by mediumTaj — COMMENTED** [review]
couple of comments and questions

File: CHANGELOG.md (hunk 3 above)
Comment (apaparazzi0329) [reply CHANGELOG.md:254]: My had removed them in node but forgot about them here

File: Source/VisualRecognitionV4/VisualRecognition.swift [hunk 9]
```diff
@@ -242,6 +261,7 @@ public class VisualRecognition {
      - parameter name: The name of the collection. The name can contain alphanumeric, underscore, hyphen, and dot
        characters. It cannot begin with the reserved prefix `sys-`.
      - parameter description: The description of the collection.
+     - parameter trainingStatus: Training status information for the collection.
```
Comment (mediumTaj) [inline Source/VisualRecognitionV4/VisualRecognition.swift:264]: should be removed

File: Source/VisualRecognitionV4/VisualRecognition.swift [hunk 10]
```diff
@@ -409,6 +437,7 @@ public class VisualRecognition {
...
```
Comment (mediumTaj) [inline Source/VisualRecognitionV4/VisualRecognition.swift:440]: should be removed

**Pull request closed without merging**